src/test/java/org/opensearch/security/dlic/rest/api/MultiTenancyConfigApiTest.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.dlic.rest.api;
+
+import org.apache.hc.core5.http.Header;
+import org.apache.hc.core5.http.HttpStatus;
+import org.junit.Test;
+
+import org.opensearch.security.support.ConfigConstants;
+import org.opensearch.security.test.helper.rest.RestHelper.HttpResponse;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.StringContains.containsString;
+
+public class MultiTenancyConfigApiTest extends AbstractRestApiUnitTest {
+
+    private static final Header ADMIN_FULL_ACCESS_USER = encodeBasicHeader("admin_all_access", "admin_all_access");
+    private static final Header USER_NO_REST_API_ACCESS = encodeBasicHeader("admin", "admin");
```

Comment:
Is this means that it is the admin user but without the `all_access` or only the `rest_api_access`?

Reply:
Yes by default `admin` has full access to the cluster and indices but not to the REST API.